.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+# Dependabot files PRs by pushing branches named
+# "dependabot/{ecosystem}/{depname}".  What it pushes to those
+# branches is pretty naive; it doesn't support us telling it that it
+# needs to run "make generate" after making an edit.
+#
+# But we can work around that by creating a workflow run "make
+# generate" on those branches!
+name: dependabot
+"on":
+  push:
+    branches:
+      - 'dependabot/**'
+
+jobs:
+  repatriate:
+    runs-on: ubuntu-latest
+    name: repatriate
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
the current version is `actions/checkout@v3` now

Reply:
Updated.